File: src/org/ggp/base/util/gdl/grammar/GdlConstant.java
package org.ggp.base.util.gdl.grammar

class GdlConstant(GdlTerm):


    value = ''

    GdlConstant(String value)
	
        self.value = value.intern()

    def String getValue()
	
        return value

    def bool isGround()
	
        return true

    def GdlSentence toSentence()
	
        return GdlPool.getProposition(this)

    def String toString()
	
        return value

File: ggpy/cruft/autocode/GdlConstant.py
#!/usr/bin/env python
""" generated source for module GdlConstant """
# package: org.ggp.base.util.gdl.grammar
@SuppressWarnings("serial")
class GdlConstant(GdlTerm):
    """ generated source for class GdlConstant """
    value = str()

    def __init__(self, value):
        """ generated source for method __init__ """
        super(GdlConstant, self).__init__()
        self.value = value.intern()

    def getValue(self):
        """ generated source for method getValue """
        return self.value

    def isGround(self):
        """ generated source for method isGround """
        return True

    def toSentence(self):
        """ generated source for method toSentence """
        return GdlPool.getProposition(self)

    def __str__(self):
        """ generated source for method toString """
        return self.value

